src/translations/translations.fr.js

```diff
@@ -671,6 +671,133 @@ export const translations = {
   'cc-jenkins-info.update.title': `Mises à jour`,
   'cc-jenkins-info.update.up-to-date': `Votre version de Jenkins est à jour.`,
   //#endregion
+  //#region cc-kv-console
+  'cc-kv-console.header': `Console`,
+  'cc-kv-console.shell.prompt': `Ligne de command`,
+  'cc-kv-console.warning': `Attention, les commandes exécutées dans la console sont envoyées telles quelles à la base de données`,
+  //#endregion
+  //#region cc-kv-explorer
+  'cc-kv-explorer.details.empty': `Sélectionnez une clé dans la liste à gauche pour en voir le détail`,
+  'cc-kv-explorer.details.unsupported': `Le type de la clé sélectionnée n'est pas supporté`,
+  'cc-kv-explorer.error.add-key': `Une erreur est survenue lors de l'ajout de la clé`,
+  'cc-kv-explorer.error.delete-key': `Une erreur est survenue lors de la suppression de la clé`,
+  'cc-kv-explorer.error.fetch-keys': `Une erreur est survenue lors de la récupération des clés`,
+  'cc-kv-explorer.error.get-key': `Une erreur est survenue lors de la récupération de la clé`,
+  'cc-kv-explorer.error.key-doesnt-exist': `La clé n'existe plus`,
+  'cc-kv-explorer.filter.all': `Tous`,
+  'cc-kv-explorer.filter.apply': `Appliquer le filtre`,
+  'cc-kv-explorer.filter.by-pattern': `Filtrer par clé`,
+  'cc-kv-explorer.filter.by-type': `Filtrer par type`,
+  'cc-kv-explorer.form.add': `Ajouter`,
+  'cc-kv-explorer.form.error.already-used': `Ce nom de clé est déjà utilisé`,
+  'cc-kv-explorer.form.key': `Nom de la clé`,
+  'cc-kv-explorer.form.reset': `Annuler`,
+  'cc-kv-explorer.form.string.value': `Valeur`,
+  'cc-kv-explorer.form.type': `Type de clé`,
+  'cc-kv-explorer.key.delete': `Supprimer la clé`,
+  'cc-kv-explorer.key.header.copy': `Copier le nom de la clé dans le presse papier`,
+  'cc-kv-explorer.key.header.delete': `Supprimer`,
+  'cc-kv-explorer.key.type.hash': `Hash`,
+  'cc-kv-explorer.key.type.list': `List`,
+  'cc-kv-explorer.key.type.set': `Set`,
+  'cc-kv-explorer.key.type.string': `String`,
+  'cc-kv-explorer.keys.empty': `La base de données est vide. Commencez par créer une clé !`,
```

Comment:
```suggestion
  'cc-kv-explorer.keys.empty': `La base de données est vide, créez une clé !`,
```

Reply:
Maybe we can go for a "- Créer une clé" here, with a link to a key creation form